packages/vscode-extension/package.json

```diff
@@ -164,6 +164,11 @@
           "name": "Deployment",
           "when": "fx-extension.isTeamsFx && fx-extension.customizedTreeview"
         },
+        {
+          "id": "teamsfx-project-and-check-upgradeV3",
+          "name": "Teams Toolkit",
```

Comment:
Is it expected that both the view and the section is called 'Teams Toolkit'?

Reply:
Thank you, it should be "UPGRADE". I think it should also be localized.